code/game/objects/items/weapons/tools/hammer.dm

```diff
@@ -152,3 +152,31 @@
 		playsound(src, 'sound/machines/hiss.ogg', 50, 0, 0)
 		user.throw_at(target, get_dist(target, user), 1, user)
 		T.stop()
+
+/obj/item/weapon/tool/hammer/ironhammer
+	name = "Tacticool Hammer  \"Tactical Hammer\"  Sledgehammer"
+	desc = "A special heavy hammer issued to IHS units for tactical breaches."
+	icon_state = "ironhammer"
+	item_state = "ironhammer"
+	wielded_icon = "ironhammer_doble"
+	force_unwielded = WEAPON_FORCE_ROBUST
+	force_wielded = WEAPON_FORCE_BRUTAL
+	armor_penetration = ARMOR_PEN_EXTREME
+	structure_damage_factor = STRUCTURE_DAMAGE_DESTRUCTIVE
+	w_class = ITEM_SIZE_SMALL
```

Comment:
You're telling me you can fit this massive sledgehammer into your pocket?
```suggestion
	w_class = ITEM_SIZE_HUGE
```

Reply:
xD
Forget to change.